code/datums/stars.dm

```diff
@@ -227,14 +227,32 @@
 	var/primary_resource
 	var/is_primary = 0
 
+	var/list/module
 
 /datum/space_station/New(var/datum/planet/P)
 	planet = P
 	SSstarmap.stations += src
+	module = list("name" = "FUCK", "Testing" = 500)
+
 
 /datum/space_station/proc/generate()
 	// TODO: Implement a more sophisticated way of generating station stocks.
+	for(var/I in SSshuttle.supply_packs)
```

Comment:
Currently this proc is getting called for every single station when SSshuttle initializes. It doesn't seem to have an impact on my start up times, but i'm not so certain how it'll hold up to the potato. Will leave this as-is for now.